packages/suite/src/utils/suite/export.ts

```diff
@@ -0,0 +1,76 @@
+import { Network } from '@wallet-types';
+
+type Field = { [key: string]: string };
+
+const CSV_SEPARATOR = ';';
+const CSV_NEWLINE = '\n';
+
+export const prepareCsv = (fields: Field, content: any[]) => {
+    const lines: string[] = [];
+
+    const fieldKeys = Object.keys(fields);
+    const fieldValues = Object.values(fields);
+
+    // Prepare header
+    let line: string[] = [];
+    fieldValues.forEach(v => {
+        line.push(v);
+    });
+
+    lines.push(line.join(CSV_SEPARATOR));
+
+    // Prepare data
+    content.forEach(c => {
+        line = [];
+
+        fieldKeys.forEach(k => {
+            line.push(c[k]);
+        });
+
+        lines.push(line.join(CSV_SEPARATOR));
+    });
+
+    return lines.join(CSV_NEWLINE);
+};
+
+export const preparePdf = (coin: Network['symbol'], fields: Field, content: any[]) => {
```

Comment:
I'd put the `coin` argument to the third place, so we are consistent with `prepareCsv`

Reply:
Addressed in ca13056.